geniza/footnotes/models.py

```diff
@@ -335,6 +339,10 @@ def formatted_display(self, extra_fields=True):
     all_authors.short_description = "Authors"
     all_authors.admin_order_field = "first_author"  # set in admin queryset
 
+    def formatted_stripped(self):
```

Comment:
don't love the method name (not very intelligible); could we just call it display for parallel with `formatted_display` ?

Reply:
Done in af1b073 👍